### PR TITLE
Codebase synchronization against jruby-1_7 

### PR DESCRIPTION
**Pull request by abstractj**

### CODE REVIEW

**Comment by vipulnsward** [comment]
@abstractj Things fail on jruby-head?

**Comment by abstractj** [comment]
@vipulnsward that was added due to the issues on Travis https://github.com/travis-ci/travis-ci/issues/1502 with RVM. Now that RVM was updated, I did the rollback.

**Comment by vipulnsward** [comment]
@abstractj Seems hmac specs are erroring out on jruby-head.

File: lib/krypt/hmac.rb [hunk 1]
```diff
@@ -1,12 +1,13 @@
 module Krypt
   class HMAC
+    include Krypt::Helper::XOR
```
Comment (vipulnsward) [inline lib/krypt/hmac.rb]: @abstractj just trying to get a hang of things here. This was replaced by Martin. You reverted this for some reason?

**Comment by abstractj** [comment]
@vipulnsward sorry, I missed that. You are correct, it doesn't make sense

Speaking about jruby-head _master_ will fail I guess for different reasons, look this is the output from _master_ https://travis-ci.org/krypt/krypt/jobs/18268779#L6565

**Pull request merged**